### PR TITLE
Adds a reference to the Opaque-ke asymmetric PAKE implementation

### PR DESCRIPTION
**Pull request by huitseeker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -873,6 +873,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 * [exonum/exonum](https://github.com/exonum/exonum) [[exonum](https://crates.io/crates/exonum)] — extensible framework for blockchain projects [<img src="https://api.travis-ci.com/exonum/exonum.svg?branch=master">](https://travis-ci.org/exonum/exonum)
 * [klutzy/suruga](https://github.com/klutzy/suruga) — A Rust implementation of [TLS 1.2](https://tools.ietf.org/html/rfc5246)
 * [libOctavo/octavo](https://github.com/libOctavo/octavo) — Modular hash and crypto library in Rust [<img src="https://api.travis-ci.org/libOctavo/octavo.svg?branch=master">](https://travis-ci.org/libOctavo/octavo)
+* [novifinancial/opaque-ke](https://github.com/novifinancial/opaque-ke) — Pure Rust implementation of the recent [OPAQUE](https://datatracker.ietf.org/doc/draft-krawczyk-cfrg-opaque/) password-authenticated key exchange.
```
Comment (palfrey) [inline README.md:876]: Please add a build status badge for the Github Actions I see in that repository. Thanks!

**Comment by huitseeker** [comment]
The link is somehow marked as non-successful in CI, but the URL is nonetheless valid:
https://github.com/novifinancial/opaque-ke/workflows/Rust%20CI/badge.svg

**Comment by palfrey** [comment]
> The link is somehow marked as non-successful in CI, but the URL is nonetheless valid:
> https://github.com/novifinancial/opaque-ke/workflows/Rust%20CI/badge.svg

The build has failed because the badge doesn't refer to a particular branch. Can you please add a branch reference to whatever the default branch is for that repo?

**Comment by huitseeker** [comment]
Thanks!

**Pull request merged**